tools/src/main/java/org/apache/rocketmq/tools/command/ha/GetSyncStateSetSubCommand.java

```diff
@@ -123,13 +123,23 @@ private void printData(String controllerAddress, List<String> brokerNames,
                     next.getKey(), next.getValue().getMasterAddress(), next.getValue().getMasterEpoch(), next.getValue().getSyncStateSetEpoch(),
                     inSyncReplicas.size());
                 for (BrokerReplicasInfo.ReplicaIdentity member : inSyncReplicas) {
-                    System.out.printf("\n InSyncReplica:\t%s\n", member.toString());
+                    System.out.printf("\n InSyncReplica:\t%s\n", member.toString() + acquireBrokerStatus(defaultMQAdminExt, member.getAddress()));
                 }
 
                 for (BrokerReplicasInfo.ReplicaIdentity member : notInSyncReplicas) {
-                    System.out.printf("\n NotInSyncReplica:\t%s\n", member.toString());
+                    System.out.printf("\n NotInSyncReplica:\t%s\n", member.toString() + acquireBrokerStatus(defaultMQAdminExt, member.getAddress()));
                 }
             }
         }
     }
+
+    private String acquireBrokerStatus(DefaultMQAdminExt defaultMQAdminExt,String brokerAddr) {
+        String status = " online";
```

Comment:
`status` should not contain format characters (like white spaces), do formatting work where it is necessary.

Reply:
i have fixed it